### DISCUSSION TITLE
bug: Not receiving Push Notifications

### DISCUSSION THREAD

**Discussion by Livijn** [ionic-team/capacitor#4342] [Q&A] [answered]
### Capacitor Version

```
  @capacitor/cli 2.4.7
  @capacitor/android 2.4.7
  @capacitor/core 2.4.7
  @capacitor/ios 2.4.7
  @capacitor/electron not installed
```

### Platform(s)

iOS

### Behavior

My app used to be able to receive Push Notifications. However, I noticed now that it has stopped working. Actually this is true for both of my apps using Push Notifications + Capacitor.  Here is the gist of my setup:

```js
PushNotifications.addListener('registration', token => {
  console.log(token.value)
});

PushNotifications.requestPermission().then(result => {
  if (result.granted) {
    PushNotifications.register();
  }
});
```

Here is what I did to test that it wasn't my server:
1. Removed app
2. Ran app from Xcode
3. Accepted permission and obtained token from the `registration` event listener.
4. Logged into Firebase -> Compose notification and entered the obtained FCM token.

Now, I don't know how to debug this further. 

### Other Technical Details

`npm --version` output: 7.6.2

`node --version` output: v15.11.0

`pod --version` output (iOS issues only): 1.10.1

**Comment by jcesarmobile** [comment]
Capacitor doesn't officially support FCM on iOS, so if you are using FCM and not receiving notifications, it's probably not related to Capacitor.
Converting the issue to discussion so maybe somebody can help you

**Comment by Livijn** [comment, marked as answer]
The reason was that my APNs Certificates had expired so I replaced them with an APNs Authentication Key.

**Comment by aabanaag** [comment]
Hello, I maybe late to this but I got the same issue, I'm not receiving anything from the console. Any help?

**Comment by SergioSuarezGil** [comment]
i have the same issue
